### ISSUE TITLE
add header and footer to fix tombstone

### ISSUE THREAD

**Issue by tacman** [survos/survos#63]
Since datatables gives us tools to search and filter, I'd like to use it, but sometimes having a bunch of columns is awkward, especially in mobile.

instead, we could have a view that puts everything in a single cell.

![Image](https://github.com/user-attachments/assets/9642ffd4-2fbe-4317-89d7-f2fb4b733249)

so that https://museado.org/en/mus/owner-browse looks like https://museado.org/en/musdig/browse (except for the columns, in datatables it would just be one column).

Using scrolller, though, those cells need to be the same height.  And the text in the middle, the description, is only a snippet.

The footer then is fixed to the bottom, with links and icons to the citation, source, etc.  

This is more important in the grid.html.twig, but defining the footer is tricky.  So we'll use the owner browse for testing.

So basically we're rendering a bootstrap card within a cell.  I guess we could also render the image there and take advantage of wrapping.

This _should_ make browsing in mobile better.  We can also reduce the image size (if it's in the cell) in mobile.

Thoughts?
